shopping-cart/shopping-cart-scala/shopping-cart/src/main/scala/com/example/shoppingcart/impl/ShoppingCartLoader.scala

```diff
@@ -35,9 +39,16 @@ abstract class ShoppingCartApplication(context: LagomApplicationContext)
   override lazy val jsonSerializerRegistry = ShoppingCartSerializerRegistry
 
   // Register the ShoppingCart persistent entity
-  persistentEntityRegistry.register(wire[ShoppingCartEntity])
+  // persistentEntityRegistry.register(wire[ShoppingCartEntity])
 
   lazy val reportRepository = wire[ShoppingCartReportRepository]
 
   readSide.register(wire[ShoppingCartReportProcessor])
+
+  clusterSharding.init(
+    Entity(
+      ShoppingCartState.typeKey,
+      entityContext => ShoppingCartState.behavior(entityContext)
+    )
+  )
```

Comment:
This replaces the `persistentEntityRegistry.register(wire[ShoppingCartEntity])`

Reply:
I never intended to leave the old one. It was a mistake. Again, the goal of this PR was to exercise the idea. Not to merge it.